lib/cli/commands/init.js

```diff
@@ -23,8 +23,10 @@ initCommand.handler = async function() {
 	const yaml = jsYaml.dump(projectConfig, {quotingType: `"`});
 
 	await writeFile(yamlPath, yaml);
-	console.log(`Wrote ui5.yaml to ${yamlPath}:\n`);
-	console.log(yaml);
+	process.stderr.write(`Wrote ui5.yaml to ${yamlPath}:`);
```

Comment:
This should remain as stdout since it's the main output of the command. (I don't mind whether to use `console.log` or `process.stdout.write`)

Reply:
Done